addons/project/static/src/views/project_task_kanban/project_task_kanban_renderer.js

```diff
@@ -12,30 +12,13 @@ export class ProjectTaskKanbanRenderer extends KanbanRenderer {
 
     }
 
-    get canMoveRecords() {
-        let canMoveRecords = super.canMoveRecords;
-        if (!canMoveRecords && this.canResequenceRecords && this.props.list.isGroupedByPersonalStages) {
-            const { groupByField } = this.props.list;
-            canMoveRecords = groupByField.readonly !== "True";
-        }
-        return canMoveRecords;
-    }
-
-    get canResequenceGroups() {
-        let canResequenceGroups = super.canResequenceGroups;
-        if (!canResequenceGroups && this.props.list.isGroupedByPersonalStages) {
-            const { groupsDraggable } = this.props.archInfo;
-            canResequenceGroups = groupsDraggable && groupsDraggable.readonly !== "True";
-        }
-        return canResequenceGroups;
-    }
-
     canCreateGroup() {
-        return (super.canCreateGroup() && this.isProjectTasksContext() && this.props.list.isGroupedByStage) || this.props.list.isGroupedByPersonalStages;
+        // This restrict the creation of project stages to the kanban view of a given project
+        return super.canCreateGroup() && (this.isProjectTasksContext() == this.props.list.isGroupedByStage);
```

Comment:
> Why not, you can also write like this:
> 
> 🙂
> 
> It is up to you, since it is equivalent. 🙂

I'm not sure, what if `this.isProjectTasksContext()` and `this.props.list.isGroupedByStage()` return false?

Reply:
Oh indeed, my bad. 🙈